minecraft_servers/byen/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "server_name": "byen",
+    "nice_name": "Byen",
+    "direct_ip": "mc.byen.net",
+    "server_wildcards": [
+      "byen.net",
+      "%.byen.net"
+    ],
+    "social": {
+      "web": "https://byen.net",
+      "web_shop": "https://butik.byen.net",
+      "web_support": "https://discord.byen.net",
```

Comment:
Discord is not supported. Only faq/web support pages are accepted for "web_support".